webpack/animateHeader.js

```diff
@@ -0,0 +1,147 @@
+import anime from 'animejs'
+
+// Detect request animation frame
+var scroll =
+  window.requestAnimationFrame ||
+  window.webkitRequestAnimationFrame ||
+  window.mozRequestAnimationFrame ||
+  window.msRequestAnimationFrame ||
+  window.oRequestAnimationFrame ||
+  // IE Fallback, you can even fallback to onscroll
+  function(callback) {
+    window.setTimeout(callback, 1000 / 60);
+  };
+var lastPosition = -1;
+
+// my Variables
+var lastSection = false;
+var headerItemTop = -1;
+var headerItemBottom = -1;
+var headerItemHeight = -1;
+
+const pinkLH = "#F70D88";
+const yellowLH = "#F9DF25";
+const blueLH = "#01ADEF";
+const whiteLH = "#FFF";
+const blackLH = "#000";
```

Comment:
Better idea: create a file that exports an object that contains those colors.

colors.js:
```js
export default {
  pink: "#F70D88",
  // ...
  black: "#000",
};
```

this file:
```js
import color from './color'

console.log(color.blue)
```